db/migrate/20200428090420_add_request_id_to_task.rb

```diff
@@ -0,0 +1,5 @@
+class AddRequestIdToTask < ActiveRecord::Migration[5.2]
+  def change
+    add_column :tasks, :x_rh_insights_request, :string, :limit => 32
```

Comment:
I got the limit by observing dev Kibana. 
It would be good if somebody can ack if it is enough

Reply:
Do we need to enforce a limit?

Reply:
right, this is varchar, not char. I'm still thinking in mysql :)